apps/challenges/views.py

```diff
@@ -3016,6 +3016,16 @@ def scale_resources_by_challenge_pk(request, challenge_pk):
         return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
 
     challenge = get_challenge_model(challenge_pk)
+    response_data = {
+        "error": "Issue with ECS."
+    }
+    return Response(response_data, status=status.HTTP_503_SERVICE_UNAVAILABLE)
```

Comment:
You are calling return here so the code after this will never get executed. Is this a indentation bug?

Reply:
I used that to test Django. Removed